docs/projects-to-work-on.md

```diff
@@ -0,0 +1,428 @@
+  [Gloire](https://github.com/streaksu/Gloire) OS distribution, which uses
+  Ironclad as its kernel.
+- **[HiRTOS](https://github.com/jgrivera67/HiRTOS)** is a high integrity RTOS
+  kernel as well as a separation kernel written in SPARK. Go and give it a try!
+  One can even install it using Alire! It currently targets the ARM Cortex-R52
+  CPU. One could try and port it to a different CPU.
+- **[Muen](https://muen.codelabs.ch/)** is a separation kernel fully written in
+  SPARK. It has been certified by some large military bodies for their internal
+  use. It also sports some drivers fully written in SPARK!
+- **[M2OS](https://m2os.unican.es/)** is a small RTOS that was created from the
+  same people behind the Ada OS [Marte](https://marte.unican.es/).
+
+There are some well known Operating Systems that allow users to run Ada programs
+in them directly even if they are written in a different language. You may want
+to use them, improve the Ada support they have or write examples or blog posts
+about their use:
+
+- **[RTEMS](https://www.rtems.org)** is a POSIX RTOS that is widely used in
+  the aerospace industry. It has official Ada support. However, its use is not
+  clearly documented nor widely known about; this could be fixed.
+
+## Embedded Programming
+
+- **[Ada Drivers Library](https://github.com/AdaCore/Ada_Drivers_Library)** is a
+  collection of drivers for microcontollers and embedded boards. You can help by
+  utilising it, testing the built-in drivers and specially by helping improve
+  drivers and boards that are not yet present.
+
+- **Create more HALs:** HAL (Hardware Abstraction Layer) is a piece of code that
+  abstracts the device away from the code. This is what generally drivers do. By
+  implementing a HAL the low level aspects of the hardware are not needed in
+  order to program the device, which makes the experience of embedded
+  programming a lot more pleasant and readable. Ada has quite a few HALs (see
+  the Ada Drivers Library), but we are nowhere near close to other communities,
+  eventhough we exceed at embedded programming.
+
+- **Improve [svd2ada](https://github.com/AdaCore/svd2ada) and
+  [startup-gen](https://github.com/AdaCore/startup-gen).** These two tools allow
+  for the quick creation of a initial HAL layer. There are a few issues and
+  limitations in their Github page which could be improved.
+
+- **Help the [SweetAda](https://github.com/gabriele-galeotti/SweetAda)
+  development framework.** SweetAda is a terrific framework designed to bring
+  Ada support to as many architectures as possible. It would greatly benefit
+  from more testing and specially documentation.
+
+- **Improve support for tasking profiles in embedded OSes (Jorvik, Ravenscar).**
+  This is a very advanced topic, but an important one. Embedded systems
+  generally do not support tasking due to the limited support that the language
+  has for them. Therefore, specialised profiles that allow for some basic,
```

Comment:
I'd say the language is not the one to support them (they are already supported through the Jorvik and Ravenscar profiles). So I'd say "implementations" or rephrase the sentence in some way, but I'm not specialist in this field, so as to propose something.